firmware/Reload Pro.cydsn/ui.c

```diff
@@ -108,13 +112,24 @@ const menudata choose_readout_menu = {
 	}
 };
 
+const menudata tools_menu_data = {
+	NULL,
+	{
+		{"Batt ISR.", {NULL, (void*)1, 0}},		

```

Comment:
I can, however 2/3rds of the existing menus dont? I felt this menu is more like the two previous to it in the file `set_readout_menu` and `choose_readout_menu` in that they use number index's instead of the `main_menu` that stores actual function pointers?